pkg/security/tls.go

```diff
@@ -48,6 +48,20 @@ func newServerTLSConfig(
 		cfg.ClientCAs = certPool
 	}
 
+	if settings.oldCipherSuitesEnabled() {
+		cfg.CipherSuites = append(
+			cfg.CipherSuites,
+			tls.TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
+			tls.TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
+			tls.TLS_RSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_RSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_RSA_WITH_AES_128_CBC_SHA,
+			tls.TLS_RSA_WITH_AES_256_CBC_SHA,
```

Comment:
Added all the removed ciphers from the previous PR in the interest of reducing the number of PRs. Since this is an opt-in flag, anyone using it will be choosing to include these in their threat model.